libs/features/components/banners/src/lib/components/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="cards">
+    <div class="container">
+    <div class="card1">
+      <p class="heading">{{ title }}</p>
+      <p class="para">{{ text }}</p>
+    </div>
+    <div>
+      <img src="{{ imgSrc }}" alt="image" class="picture" />
+    </div>
+  </div>
+</div>
```

Comment:
The positioning of this elements is dynamic. I don't see any code here to implement that i.e image placement.